DEBUG - 2025-04-18 01:42:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 01:42:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 01:42:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 01:42:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 01:42:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 01:42:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 01:43:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 01:43:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:12:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:12:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:15:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:15:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:19:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:19:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:20:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:24:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:25:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:25:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:27:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:29:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:29:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:30:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:30:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:30:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:31:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:32:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:32:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:32:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:33:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:41:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 02:41:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-04-18 03:28:50 --> mysqli_sql_exception: Table 'todo1.users' doesn't exist in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`....', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`....')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`....')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`....', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp1\htdocs\listoria\app\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#10 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 03:28:50 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'todo1.users' doesn't exist
[Method: GET, Route: tugas/detail/2]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:28:50 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'todo1.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:28:50 --> [Caused by] mysqli_sql_exception: Table 'todo1.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 03:30:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 03:30:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 03:32:29 --> ErrorException: Cannot redeclare App\Models\TugasModel::getTugasWithDetails()
[Method: GET, Route: tugas/detail/2]
in APPPATH\Models\TugasModel.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-04-18 03:34:34 --> CodeIgniter\Exceptions\BadMethodCallException: Call to undefined method App\Models\AttachmentModel::getAttachments
[Method: GET, Route: tugas/detail/2]
in SYSTEMPATH\Model.php on line 930.
 1 APPPATH\Controllers\Tugas.php(110): CodeIgniter\Model->__call('getAttachments', [...])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 03:41:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 03:44:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-04-18 03:44:04 --> mysqli_sql_exception: Unknown column 'shared.user_id' in 'where clause' in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`....', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`....')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`....')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`....', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp1\htdocs\listoria\app\Models\TugasModel.php(25): CodeIgniter\BaseModel->findAll()
#7 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(21): App\Models\TugasModel->getSharedTugasByUser('2')
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->index()
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#10 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 03:44:04 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'shared.user_id' in 'where clause'
[Method: GET, Route: tugas]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = :shared.user_id:', [...], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\TugasModel.php(25): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\Tugas.php(21): App\Models\TugasModel->getSharedTugasByUser('2')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:44:04 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'shared.user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = :shared.user_id:', [...], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Models\TugasModel.php(25): CodeIgniter\BaseModel->findAll()
 7 APPPATH\Controllers\Tugas.php(21): App\Models\TugasModel->getSharedTugasByUser('2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:44:04 --> [Caused by] mysqli_sql_exception: Unknown column 'shared.user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = :shared.user_id:', [...], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Models\TugasModel.php(25): CodeIgniter\BaseModel->findAll()
 8 APPPATH\Controllers\Tugas.php(21): App\Models\TugasModel->getSharedTugasByUser('2')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 03:45:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-04-18 03:45:04 --> mysqli_sql_exception: Unknown column 'shared.tugas_id' in 'on clause' in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`....', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`....')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`....')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`....', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp1\htdocs\listoria\app\Models\TugasModel.php(25): CodeIgniter\BaseModel->findAll()
#7 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(21): App\Models\TugasModel->getSharedTugasByUser('2')
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->index()
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#10 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 03:45:04 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'shared.tugas_id' in 'on clause'
[Method: GET, Route: tugas]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = :shared.id_user:', [...], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\TugasModel.php(25): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\Tugas.php(21): App\Models\TugasModel->getSharedTugasByUser('2')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:45:04 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'shared.tugas_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = :shared.id_user:', [...], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Models\TugasModel.php(25): CodeIgniter\BaseModel->findAll()
 7 APPPATH\Controllers\Tugas.php(21): App\Models\TugasModel->getSharedTugasByUser('2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:45:04 --> [Caused by] mysqli_sql_exception: Unknown column 'shared.tugas_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = :shared.id_user:', [...], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Models\TugasModel.php(25): CodeIgniter\BaseModel->findAll()
 8 APPPATH\Controllers\Tugas.php(21): App\Models\TugasModel->getSharedTugasByUser('2')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 03:45:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 03:45:31 --> ErrorException: Undefined variable $pager
[Method: GET, Route: tugas]
in APPPATH\Views\tugas\index.php on line 32.
 1 APPPATH\Views\tugas\index.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $pager', 'C:\\xampp1\\htdocs\\listoria\\app\\Views\\tugas\\index.php', 32)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp1\\htdocs\\listoria\\app\\Views\\tugas\\index.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('tugas/index', [], true)
 5 APPPATH\Controllers\Tugas.php(23): view('tugas/index', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 03:50:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-04-18 03:50:07 --> mysqli_sql_exception: Unknown column 'shared.user_id' in 'where clause' in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
#6 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
#7 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->index()
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-04-18 03:50:07 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'shared.user_id' in 'where clause'
[Method: GET, Route: tugas]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = :shared.user_id:', [...], false)
 2 SYSTEMPATH\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
 3 SYSTEMPATH\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
 4 APPPATH\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:50:07 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'shared.user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = :shared.user_id:', [...], false)
 4 SYSTEMPATH\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
 5 SYSTEMPATH\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
 6 APPPATH\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:50:07 --> [Caused by] mysqli_sql_exception: Unknown column 'shared.user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = :shared.user_id:', [...], false)
 5 SYSTEMPATH\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
 6 SYSTEMPATH\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
 7 APPPATH\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 03:50:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-04-18 03:50:29 --> mysqli_sql_exception: Unknown column 'shared.user_id' in 'where clause' in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
#6 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
#7 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->index()
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-04-18 03:50:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'shared.user_id' in 'where clause'
[Method: GET, Route: tugas]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = :shared.user_id:', [...], false)
 2 SYSTEMPATH\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
 3 SYSTEMPATH\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
 4 APPPATH\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:50:29 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'shared.user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = :shared.user_id:', [...], false)
 4 SYSTEMPATH\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
 5 SYSTEMPATH\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
 6 APPPATH\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:50:29 --> [Caused by] mysqli_sql_exception: Unknown column 'shared.user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = :shared.user_id:', [...], false)
 5 SYSTEMPATH\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
 6 SYSTEMPATH\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
 7 APPPATH\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 03:50:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-04-18 03:50:30 --> mysqli_sql_exception: Unknown column 'shared.user_id' in 'where clause' in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
#6 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
#7 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->index()
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-04-18 03:50:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'shared.user_id' in 'where clause'
[Method: GET, Route: tugas]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = :shared.user_id:', [...], false)
 2 SYSTEMPATH\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
 3 SYSTEMPATH\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
 4 APPPATH\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:50:30 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'shared.user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = :shared.user_id:', [...], false)
 4 SYSTEMPATH\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
 5 SYSTEMPATH\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
 6 APPPATH\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:50:30 --> [Caused by] mysqli_sql_exception: Unknown column 'shared.user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`id_tugas` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`user_id` = :shared.user_id:', [...], false)
 5 SYSTEMPATH\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
 6 SYSTEMPATH\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
 7 APPPATH\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 03:51:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-04-18 03:51:12 --> mysqli_sql_exception: Unknown column 'shared.tugas_id' in 'on clause' in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
#6 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
#7 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->index()
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-04-18 03:51:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'shared.tugas_id' in 'on clause'
[Method: GET, Route: tugas]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = :shared.id_user:', [...], false)
 2 SYSTEMPATH\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
 3 SYSTEMPATH\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
 4 APPPATH\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:51:12 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'shared.tugas_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = :shared.id_user:', [...], false)
 4 SYSTEMPATH\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
 5 SYSTEMPATH\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
 6 APPPATH\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:51:12 --> [Caused by] mysqli_sql_exception: Unknown column 'shared.tugas_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `tugas`
JOIN `shared` ON `shared`.`tugas_id` = `tugas`.`id`
JOIN `user` ON `user`.`id_user` = `tugas`.`creator_id`
WHERE `shared`.`id_user` = :shared.id_user:', [...], false)
 5 SYSTEMPATH\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(false)
 6 SYSTEMPATH\BaseModel.php(1283): CodeIgniter\Model->countAllResults(false)
 7 APPPATH\Controllers\Tugas.php(26): CodeIgniter\BaseModel->paginate(10)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 03:51:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 03:51:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 03:54:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 03:54:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 03:54:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-04-18 03:54:51 --> mysqli_sql_exception: Table 'todo1.users' doesn't exist in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`....', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`....')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`....')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`....', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp1\htdocs\listoria\app\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#10 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 03:54:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'todo1.users' doesn't exist
[Method: GET, Route: tugas/detail/2]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:54:51 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'todo1.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 03:54:51 --> [Caused by] mysqli_sql_exception: Table 'todo1.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-18 05:01:05 --> mysqli_sql_exception: Table 'todo1.users' doesn't exist in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`....', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`....')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`....')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`....', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp1\htdocs\listoria\app\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#10 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 05:01:05 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'todo1.users' doesn't exist
[Method: GET, Route: tugas/detail/2]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 05:01:05 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'todo1.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 05:01:05 --> [Caused by] mysqli_sql_exception: Table 'todo1.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `users`.`nama` as `creator_name`
FROM `tugas`
JOIN `users` ON `users`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-18 05:02:19 --> mysqli_sql_exception: Unknown column 'user.nama' in 'field list' in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`....', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`....')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`....')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`....', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp1\htdocs\listoria\app\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#10 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 05:02:19 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user.nama' in 'field list'
[Method: GET, Route: tugas/detail/2]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`nama` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 05:02:19 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user.nama' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `user`.`nama` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `user`.`nama` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`nama` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 05:02:19 --> [Caused by] mysqli_sql_exception: Unknown column 'user.nama' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`.*, `user`.`nama` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `user`.`nama` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `user`.`nama` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`nama` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-18 05:02:34 --> mysqli_sql_exception: Unknown column 'user.id' in 'on clause' in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`....', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`....')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`....')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`....', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp1\htdocs\listoria\app\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#10 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 05:02:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user.id' in 'on clause'
[Method: GET, Route: tugas/detail/2]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 05:02:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user.id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 05:02:34 --> [Caused by] mysqli_sql_exception: Unknown column 'user.id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-18 05:03:20 --> mysqli_sql_exception: Unknown column 'user.id' in 'on clause' in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`....', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`....')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`....')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`....', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp1\htdocs\listoria\app\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#10 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 05:03:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user.id' in 'on clause'
[Method: GET, Route: tugas/detail/2]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 05:03:20 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user.id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 05:03:20 --> [Caused by] mysqli_sql_exception: Unknown column 'user.id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-18 05:03:21 --> mysqli_sql_exception: Unknown column 'user.id' in 'on clause' in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`....', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`....')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`....')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`....', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp1\htdocs\listoria\app\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#10 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 05:03:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user.id' in 'on clause'
[Method: GET, Route: tugas/detail/2]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 05:03:21 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user.id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 05:03:21 --> [Caused by] mysqli_sql_exception: Unknown column 'user.id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = \'2\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tugas`.*, `user`.`username` as `creator_name`
FROM `tugas`
JOIN `user` ON `user`.`id` = `tugas`.`creator_id`
WHERE `tugas`.`id` = :tugas.id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\TugasModel.php(23): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\Tugas.php(103): App\Models\TugasModel->getTugasWithDetails('2')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('2')
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 05:04:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 05:04:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 05:45:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 05:45:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 05:45:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 05:46:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 05:51:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 05:53:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 05:56:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 05:56:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 05:56:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:00:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:01:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:01:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:01:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:01:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:01:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:01:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:01:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:01:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:01:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:01:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:03:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:04:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:04:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:05:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:05:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 06:10:10 --> ParseError: syntax error, unexpected token "..."
[Method: GET, Route: tugas/detail/4]
in APPPATH\Controllers\Tugas.php on line 128.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp1\\htdocs\\listoria\\app\\Controllers\\tugas.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\tugas')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\tugas')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\tugas', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 06:11:05 --> ParseError: Unclosed '{' on line 9
[Method: GET, Route: tugas/detail/4]
in APPPATH\Controllers\Tugas.php on line 142.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp1\\htdocs\\listoria\\app\\Controllers\\tugas.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\tugas')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\tugas')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\tugas', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 06:11:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-04-18 06:11:27 --> mysqli_sql_exception: Unknown column 'tugas_id' in 'where clause' in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp1\htdocs\listoria\app\Models\TugasModel.php(36): CodeIgniter\BaseModel->findAll()
#7 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('4')
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->detail('4')
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#10 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 06:11:27 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'tugas_id' in 'where clause'
[Method: GET, Route: tugas/detail/4]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = :tugas_id:', [...], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\TugasModel.php(36): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('4')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('4')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 06:11:27 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'tugas_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'4\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'4\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = :tugas_id:', [...], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Models\TugasModel.php(36): CodeIgniter\BaseModel->findAll()
 7 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('4')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('4')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 06:11:27 --> [Caused by] mysqli_sql_exception: Unknown column 'tugas_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'4\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'4\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'4\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = :tugas_id:', [...], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Models\TugasModel.php(36): CodeIgniter\BaseModel->findAll()
 8 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('4')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('4')
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 06:14:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-04-18 06:14:19 --> mysqli_sql_exception: Unknown column 'tugas_id' in 'where clause' in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp1\htdocs\listoria\app\Models\TugasModel.php(36): CodeIgniter\BaseModel->findAll()
#7 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('4')
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->detail('4')
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#10 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 06:14:19 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'tugas_id' in 'where clause'
[Method: GET, Route: tugas/detail/4]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = :tugas_id:', [...], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\TugasModel.php(36): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('4')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('4')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 06:14:19 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'tugas_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'4\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'4\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = :tugas_id:', [...], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Models\TugasModel.php(36): CodeIgniter\BaseModel->findAll()
 7 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('4')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('4')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 06:14:19 --> [Caused by] mysqli_sql_exception: Unknown column 'tugas_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'4\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'4\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'4\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = :tugas_id:', [...], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Models\TugasModel.php(36): CodeIgniter\BaseModel->findAll()
 8 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('4')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('4')
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 06:14:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:14:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:14:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:14:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:14:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-04-18 06:14:55 --> mysqli_sql_exception: Unknown column 'tugas_id' in 'where clause' in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp1\htdocs\listoria\app\Models\TugasModel.php(36): CodeIgniter\BaseModel->findAll()
#7 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('3')
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->detail('3')
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#10 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 06:14:55 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'tugas_id' in 'where clause'
[Method: GET, Route: tugas/detail/3]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = :tugas_id:', [...], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\TugasModel.php(36): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('3')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('3')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 06:14:55 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'tugas_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'3\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = :tugas_id:', [...], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Models\TugasModel.php(36): CodeIgniter\BaseModel->findAll()
 7 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('3')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('3')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 06:14:55 --> [Caused by] mysqli_sql_exception: Unknown column 'tugas_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'3\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'3\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = :tugas_id:', [...], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Models\TugasModel.php(36): CodeIgniter\BaseModel->findAll()
 8 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('3')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('3')
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 06:15:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:15:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:15:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:15:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:15:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:15:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:53:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:53:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:53:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:53:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:55:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:55:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:55:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:56:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:56:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 06:56:07 --> CodeIgniter\Exceptions\BadMethodCallException: Call to undefined method App\Models\TugasModel::isUserAdmin
[Method: GET, Route: tugas/detail/3]
in SYSTEMPATH\Model.php on line 930.
 1 APPPATH\Controllers\Tugas.php(129): CodeIgniter\Model->__call('isUserAdmin', [...])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('3')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 06:58:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-04-18 06:58:01 --> mysqli_sql_exception: Unknown column 'tugas_id' in 'where clause' in C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\listoria\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp1\htdocs\listoria\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp1\htdocs\listoria\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp1\htdocs\listoria\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp1\htdocs\listoria\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp1\htdocs\listoria\app\Models\TugasModel.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp1\htdocs\listoria\app\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('3')
#8 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(933): App\Controllers\Tugas->detail('3')
#9 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
#10 C:\xampp1\htdocs\listoria\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp1\htdocs\listoria\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp1\htdocs\listoria\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp1\htdocs\listoria\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 06:58:01 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'tugas_id' in 'where clause'
[Method: GET, Route: tugas/detail/3]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = :tugas_id:', [...], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\TugasModel.php(31): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('3')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('3')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 06:58:01 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'tugas_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'3\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = :tugas_id:', [...], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Models\TugasModel.php(31): CodeIgniter\BaseModel->findAll()
 7 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('3')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('3')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 06:58:01 --> [Caused by] mysqli_sql_exception: Unknown column 'tugas_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'3\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `attachment`
WHERE `tugas_id` = \'3\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `attachment`
WHERE `tugas_id` = :tugas_id:', [...], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Models\TugasModel.php(31): CodeIgniter\BaseModel->findAll()
 8 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('3')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('3')
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 06:58:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:58:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:58:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:58:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 06:59:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:00:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:00:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:00:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:00:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:08:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:08:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:08:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:14:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:14:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:14:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:15:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:15:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:15:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:16:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:16:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:16:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:16:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:16:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:16:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:16:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:16:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:19:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:19:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:19:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:19:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:19:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:19:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:19:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:20:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:20:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:20:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:20:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:20:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:20:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:20:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:20:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:21:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:21:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:21:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:21:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:21:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:21:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:21:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:21:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:22:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:22:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:23:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:23:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:23:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:23:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:23:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:28:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:29:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:29:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:30:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:30:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:31:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:31:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:31:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 07:31:39 --> ParseError: Unclosed '{' on line 9
[Method: GET, Route: tugas]
in APPPATH\Controllers\Tugas.php on line 142.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp1\\htdocs\\listoria\\app\\Controllers\\tugas.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\tugas')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\tugas')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\tugas', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 07:35:48 --> ParseError: Unclosed '{' on line 9
[Method: GET, Route: tugas]
in APPPATH\Controllers\Tugas.php on line 142.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp1\\htdocs\\listoria\\app\\Controllers\\tugas.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\tugas')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\tugas')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\tugas', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 07:36:25 --> ParseError: Unclosed '{' on line 9
[Method: GET, Route: tugas]
in APPPATH\Controllers\Tugas.php on line 142.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp1\\htdocs\\listoria\\app\\Controllers\\tugas.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\tugas')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\tugas')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\tugas', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 07:36:46 --> ParseError: syntax error, unexpected token "protected", expecting end of file
[Method: GET, Route: tugas]
in APPPATH\Controllers\Tugas.php on line 10.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp1\\htdocs\\listoria\\app\\Controllers\\tugas.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\tugas')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\tugas')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\tugas', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 07:37:06 --> ParseError: syntax error, unexpected token "public", expecting end of file
[Method: GET, Route: tugas]
in APPPATH\Controllers\Tugas.php on line 12.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp1\\htdocs\\listoria\\app\\Controllers\\tugas.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\tugas')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\tugas')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\tugas', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 07:38:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:38:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:38:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:38:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:38:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:38:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:38:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:38:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:38:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:38:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:38:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:38:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:38:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:45:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:46:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:46:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:52:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:55:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:56:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:57:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:57:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 07:57:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:00:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:01:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:01:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:01:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:02:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:02:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:03:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:03:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:03:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:04:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:04:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:04:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:04:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:04:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:06:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:08:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:08:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:10:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:10:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:15:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 08:15:16 --> Error: Class "App\Models\AttachmentModel" not found
[Method: GET, Route: tugas/detail/7]
in APPPATH\Models\TugasModel.php on line 30.
 1 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('7')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('7')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 08:16:17 --> ParseError: syntax error, unexpected token "<", expecting end of file
[Method: GET, Route: tugas/detail/7]
in APPPATH\Models\TugasModel.php on line 5.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp1\\htdocs\\listoria\\app\\Models\\TugasModel.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\TugasModel')
 3 APPPATH\Controllers\Tugas.php(14): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\TugasModel')
 4 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Tugas->__construct()
 5 SYSTEMPATH\CodeIgniter.php(498): CodeIgniter\CodeIgniter->createController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 08:17:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 08:17:13 --> Error: Class "App\Models\AttachmentModel" not found
[Method: GET, Route: tugas/detail/7]
in APPPATH\Models\TugasModel.php on line 29.
 1 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('7')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('7')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 08:18:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 08:18:59 --> Error: Class "App\Models\AttachmentModel" not found
[Method: GET, Route: tugas/detail/7]
in APPPATH\Models\TugasModel.php on line 29.
 1 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('7')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('7')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 08:19:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 08:19:00 --> Error: Class "App\Models\AttachmentModel" not found
[Method: GET, Route: tugas/detail/7]
in APPPATH\Models\TugasModel.php on line 29.
 1 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('7')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('7')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 08:20:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 08:20:18 --> Error: Class "App\Models\AttachmentModel" not found
[Method: GET, Route: tugas/detail/7]
in APPPATH\Models\TugasModel.php on line 29.
 1 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('7')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('7')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 08:20:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 08:20:19 --> Error: Class "App\Models\AttachmentModel" not found
[Method: GET, Route: tugas/detail/7]
in APPPATH\Models\TugasModel.php on line 29.
 1 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('7')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('7')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 08:20:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 08:20:20 --> Error: Class "App\Models\AttachmentModel" not found
[Method: GET, Route: tugas/detail/7]
in APPPATH\Models\TugasModel.php on line 29.
 1 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('7')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('7')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 08:20:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 08:20:37 --> Error: Class "App\Models\AttachmentModel" not found
[Method: GET, Route: tugas/detail/7]
in APPPATH\Models\TugasModel.php on line 29.
 1 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('7')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('7')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 08:20:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 08:20:38 --> Error: Class "App\Models\AttachmentModel" not found
[Method: GET, Route: tugas/detail/7]
in APPPATH\Models\TugasModel.php on line 29.
 1 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('7')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('7')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 08:20:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 08:20:39 --> Error: Class "App\Models\AttachmentModel" not found
[Method: GET, Route: tugas/detail/7]
in APPPATH\Models\TugasModel.php on line 29.
 1 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('7')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('7')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 08:20:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 08:20:40 --> Error: Class "App\Models\AttachmentModel" not found
[Method: GET, Route: tugas/detail/7]
in APPPATH\Models\TugasModel.php on line 29.
 1 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('7')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('7')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 08:22:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 08:22:09 --> Error: Class "App\Models\AttachmentModel" not found
[Method: GET, Route: tugas/detail/7]
in APPPATH\Models\TugasModel.php on line 31.
 1 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('7')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('7')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 08:29:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:29:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:29:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-18 08:29:12 --> Error: Class "App\Models\AttachmentModel" not found
[Method: GET, Route: tugas/detail/7]
in APPPATH\Models\TugasModel.php on line 31.
 1 APPPATH\Controllers\Tugas.php(128): App\Models\TugasModel->getAttachments('7')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tugas->detail('7')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tugas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-04-18 08:29:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-18 08:29:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
